shinydashboard/lantern/functions/endpoints.R

```diff
@@ -165,17 +166,40 @@ get_fhir_resource_count <- function(fhir_resources_tbl) {
 
 get_capstat_fields_count <- function(capstat_fields_tbl) {
   res <- capstat_fields_tbl %>%
-    group_by(field, exist, fhir_version) %>%
+    group_by(field, exist, fhir_version, extension) %>%
     count() %>%
     filter(exist == "true") %>%
+    filter(extension == "false") %>%
     ungroup() %>%
     select(-exist) %>%
+    select(-extension) %>%
+    rename(Fields = field, Endpoints = n)
+}
+
+get_capstat_extensions_count <- function(capstat_fields_tbl) {
```

Comment:
could you just pass the extension value you want through as a parameter to use in the filter function so these wouldn't have to be two separate functions?